guidelines/application_requirements.md

```diff
@@ -0,0 +1,84 @@
+# Guidelines for Application Requirements
+
+Authors: chasm@google.com, jlewi@google.com, kam.d.kasravi@intel.com
+
+## Objective
+
+Define requirements and recommendations aimed at graduating applications (e.g. controller and web apps) in Kubeflow to 1.0.
+
+## Scope
+
+This document is intended to cover applications that are deployed in the Kubernetes cluster and in particular:
+
+1. Custom resources and controllers
+1. Web applications
+
+This document is not intended to cover SDKs (e.g. fairing) or client side tools (like kfctl).
+
+## Goals
+
+The purpose of this doc is to:
+
+1. Establish criterion for graduating a Kubeflow application to a supported Kubeflow application; i.e. 1.0
+1. Set clear expectations for users regarding Kubeflow applications that are 1.0
```

Comment:
Same here - are we setting expectations for applications that are 1.0, or those that are compliant with Kubeflow 1.0+ standards?